app/models/search_timdex.rb

```diff
@@ -0,0 +1,35 @@
+# Searches TIMDEX! API and formats results into {Result} object
+#
+# == Required Environment Variables:
+# - TIMDEX_URL
+#
+# == Optional Environment Variables:
+# - RESULTS_PER_BOX
+#
+# == See Also
+# - https://timdex.mit.edu/playground
+class SearchTimdex
+  attr_reader :results
+
+  TIMDEX_URL = ENV['TIMDEX_URL'].freeze
+
+  def initialize
+    @timdex_http = HTTP.persistent(TIMDEX_URL)
+    @results = {}
+  end
+
+  # Run a search
+  # @param term [string] The string we are searching for
+  # @return [Hash] A Hash with search metadata and an Array of {Result}s
+  def search(term)
+    @query = '{"query":"{\n  search(searchterm: \"' + term + '\", source: \"MIT ArchivesSpace\") {\nhits\n    records {\n      sourceLink\n      title\n    }\n  }\n}\n"}'
```

Comment:
I am aware I stated I didn't think we needed a client library for this simple GraphQL query... however, I sort of disagree with myself now that I see this.

I had looked at a nice GraphQL library GitHub provides in this experimental work a while back and I feel like it may be worth it. If you have a bit of time maybe take a look at what I did and think about whether you think it might make this more readable / maintainable over time:

https://github.com/MITLibraries/timlight/pull/6/files#diff-c2795793c537f0d56af3be0de530a0adR1

Reply:
Following up on this - I haven't tried adding the gem, but some of the syntax that I was using here turns out not to be needed. Specifically, the encoding of newlines is superfluous. Further commits on this method are now available.

Reply:
Let's continue with your initial approach to completion of the work. We can consider a refactor around this in the future. I'll update the related Jira ticket to reflect this.